### DISCUSSION TITLE
knplabs/doctrine-behaviors -> gedmo/doctrine-extensions

### DISCUSSION THREAD

**Discussion by alcohol** [rectorphp/rector#8551] [Q&A]
I noticed you wrote several rectors to migrate from Knp to Gedmo in the past. Any chance of getting some help to write a reverse set? Knp has not seen a release since 2019. And while some comments in various issues on the repository suggest that might change some time in the future, I would much rather convert all of my clients Knp Translatable use to Gedmo, since they already use Gedmo for everything else. They stuck to Knp for translations because of https://github.com/a2lix/TranslationFormBundle -- but rarely actually use that now.

**Comment by TomasVotruba** [comment]
Hi, this usually takes a private project to invest the resources to make it happen.
We can help with it as a service https://getrector.com/hire-team
